test/test_subject_age.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+import pytest
+from bids2openminds.main import create_openminds_age
+
+example_ages = [("89+", "QuantitativeValueRange"),
+                (45, "QuantitativeValue"), ("XX", None)]
+
+
+@ pytest.mark.parametrize("age,type", example_ages)
+def test_subject_age(age, type):
+    data_subject_table = pd.DataFrame(data={'age': [age]})
+    openminds_age = create_openminds_age(data_subject_table)
+    match type:
```

Comment:
The `match...case` feature was introduced in Python 3.10. bids2openminds currently claims to support Python 3.9+

I think we can merge this for now, but we should discuss whether we want to maintain Python 3.9 compatibility (numpy dropped it last month); if so this can then be replaced with a plain `if type == "QuantitativeValueRange": ...`